packages/front-end/components/Experiment/TabbedPage/HealthTab.tsx

```diff
@@ -23,33 +24,32 @@ export default function HealthTab({
   onDrawerNotify,
   onSnapshotUpdate,
 }: Props) {
-  const { error, snapshot, phase } = useSnapshot();
+  const {
+    error,
+    snapshot,
+    phase,
+    mutateSnapshot,
+    setAnalysisSettings,
+  } = useSnapshot();
   const { runHealthTrafficQuery } = useOrgSettings();
-  const { apiCall } = useAuth();
   const { refreshOrganization } = useUser();
   const permissions = usePermissions();
   const hasPermissionToEditOrgSettings = permissions.check(
     "organizationSettings"
   );
   const [healthIssues, setHealthIssues] = useState<IssueValue[]>([]);
   // Clean up notification counter & health issues before unmounting
+
+  const [uiMode, setUiMode] = useState<"open" | "setup">("open");
```

Comment:
Is this just used for the open state of the modal? If so maybe name this something like `setupModalOpen` and make it a boolean instead of a string

Reply:
done, ty.